test/test_integration.py

```diff
@@ -382,8 +382,8 @@ def socket_handler(listener):
             # We should get two packets: one connection header string, one
             # SettingsFrame. Rather than respond to the packets, send a GOAWAY
             # frame with error code 0 indicating clean shutdown.
-            first = sock.recv(65535)
-            second = sock.recv(65535)
+            first = sock.recv(65535)  # pragma: no flakes
```

Comment:
The `# prgama: no flakes` make ignore that line. An error happens if it removed as below.

```
hyper/test/test_integration.py:385: UnusedVariable
local variable 'first' is assigned to but never used
```
See also `Ignoring certain lines in files` on https://pypi.python.org/pypi/pytest-flakes